lib/routes/home/widgets/app_bar/home_app_bar.dart

```diff
@@ -24,7 +24,10 @@ class HomeAppBar extends AppBar {
               "src/icon/hamburger.svg",
               height: 24.0,
               width: 24.0,
-              color: themeData.appBarTheme.actionsIconTheme?.color,
+              colorFilter: ColorFilter.mode(
+                themeData.appBarTheme.actionsIconTheme!.color!,
```

Comment:
Are there any chance that these can be null ?

Reply:
Not in our case, they are defined on every theme available on our app.

Reply:
Great, thank you 🙏